similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/AbstractPatientSimilarityView.java

```diff
@@ -130,6 +130,27 @@ public DocumentReference getReporter()
     }
 
     @Override
+    public String getOwnerName()
+    {
+        // TODO: Update once there is a convenient method for accessing the owner of the patient record.
```

Comment:
Hmm, I was expecting us to have a way to access additional patient information (e.g. Owner) in a more privacy conscious way. Right now, you have to be careful about having references to PatientAccess objects lying around since they expose the ability to set the Owner as well. The only permissions-aware interface is a ScriptService.